### PR TITLE
build ArmVirtPkg for arm64-usr

### PR DESCRIPTION
**Pull request by mischief**
this builds QEMU_EFI.fd for the purpose of running coreos images in qemu-system-aarch64.

### CODE REVIEW

**Comment by mischief** [comment]
cc @marineam @glevand

File: sys-firmware/edk2-armvirt/edk2-armvirt-99999.ebuild [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+	S="${WORKDIR}/${MY_P}"
+	SRC_URI="http://storage.core-os.net/mirror/snapshots/${MY_P}.tar.xz"
+	KEYWORDS="-* arm64"
+fi
+
+OPENSSL_PV="1.0.2d"
+OPENSSL_P="openssl-${OPENSSL_PV}"
+SRC_URI+=" mirror://openssl/source/${OPENSSL_P}.tar.gz"
+
+DEPEND=">=dev-lang/nasm-2.0.7"
```
Comment (marineam) [inline sys-firmware/edk2-armvirt/edk2-armvirt-99999.ebuild]: I would assume nasm isn't used in the arm build, is it?

**Comment by mischief** [comment]
hm, afaict, no.

**Comment by mischief** [comment]
squashed removal of iasl and nasm.

**Comment by marineam** [comment]
LGTM

**Comment by glevand** [comment]
I tested the resulting QEMU_EFI.fd and it seems to work OK.

**Pull request merged**